Description: Avoid network access when building docs
 Upstream is using sphinx.ext.intersphinx, disabling it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-14

--- ironic-2014.1~b3.orig/doc/source/conf.py
+++ ironic-2014.1~b3/doc/source/conf.py
@@ -6,7 +6,6 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.viewcode',
               'sphinxcontrib.httpdomain',
               'sphinxcontrib.pecanwsme.rest',
